### PR TITLE
Reveal folds when using jump maps in commits

### PR DESCRIPTION
**Pull request by odnoletkov**

### CODE REVIEW

File: autoload/fugitive.vim [hunk 1]
```diff
@@ -2731,6 +2736,7 @@ function! s:PreviousItem(count) abort
     endif
   endfor
   call s:StageReveal()
+  exe 'normal! zv'
```
Comment (tpope) [inline autoload/fugitive.vim]: Looks like it'd be less repetitive to do this inside `s:StageReveal()`?

**Comment by odnoletkov** [comment]
Apparently this also resolves the issue with jump maps sometimes hanging in commits. Conditions in the scrolling loop in `StageReveal` didn't always work properly with folds.

File: autoload/fugitive.vim [hunk 2]
```diff
@@ -2629,6 +2629,7 @@ function! s:Do(action, visual) abort
 endfunction
 
 function! s:StageReveal(...) abort
+  exe 'normal! zv'
   let begin = a:0 ? a:1 : line('.')
```
Comment (tpope) [inline autoload/fugitive.vim]: "Open fold on a current line" sounds like the wrong behavior when acting on a line number argument.

Comment (odnoletkov) [reply autoload/fugitive.vim]: I don't see the line number arg version used anywhere. What are the sample use cases for it? I'm trying to understand if the arg version should even reveal the fold.

Also, is it even possible to reveal the folded line without jumping to it first?

Comment (tpope) [reply autoload/fugitive.vim]: Oh, perfect. Go ahead and rip out the argument entirely if you don't mind and this is good to go.

**Pull request merged**